### ISSUE TITLE
Update BigQuery connector with Spark for the new Dataproc Image 'dataproc-2.1.x version'

### ISSUE THREAD

**Issue by yuyatinnefeld** [GoogleCloudDataproc/initialization-actions#1044]
Currently is the BigQuery Spark connector `./connectors/connectors.sh` not able for the new Dataproc image. 

Can somebody update the connectors.sh for the image? 

```bash
  if [[ $(min_version "$DATAPROC_VERSION" 2.0) == 2.0 ]]; then
    local -r hadoop_version_suffix=hadoop3
  else
    local -r hadoop_version_suffix=hadoop2
  fi
```

Thanks a lot.

**Comment by yuyatinnefeld** [comment]
thanks @khaledh for the great information! This worked as you mentioned❤️

**Comment by khaledh** [comment]
This has been fixed in #1064. Thanks @cjac!
